Examples/SubscriberExample/Sources/Screens/Map/MapViewController.swift

```diff
@@ -218,10 +218,15 @@ extension MapViewController: SubscriberDelegate {
         self.location = location
         scrollToReceivedLocation()
     }
+    
+    func subscriber(sender: Subscriber, didUpdateRawLocation location: Location) {
+        // TODO
+    }
```

Comment:
Why is it added here? :thinking: I'd remove it if it's not necessary :wink:

Reply:
as I mentioned in other PR where you ask the same, there is no `optional` methods in Swift protocols, but I can create an extension with default implementation of this methos. This will work as an `optional`.

changed here 768614547668ffcd78a5b0bca650072e60b93f19